.eslintrc

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
@michaelmoney Why eslint? I'm not sure is it gonna work but as we've desided to use typescript, seems tslint.json is a proper way :)

Reply:
Sorry, forgot to add it. Tslint is going to be deprecated this year and eslint will be used both for JS and TS. I thought we can be ready for this transition. 
>Palantir is the creator and primary maintainer of TSLint, the standard linter for the TypeScript programming language. As the TypeScript community is working toward a unified developer experience across the TypeScript and JavaScript languages, we are committed to support the convergence of TSLint and ESLint; in this blog post we explain the Why and How of our efforts.

More here:
https://medium.com/palantir/tslint-in-2019-1a144c2317a9
and here: https://github.com/typescript-eslint/typescript-eslint

That's why I followed this setup, but forgot to add tslint dependency and configuration in my PR. 
https://github.com/piotrwitek/react-redux-typescript-scripts

Let me know what do you think about it.

Reply:
LGTM, thanks for sharing this 👍